packages/grive2-git/grive2-git.pacscript

```diff
@@ -0,0 +1,33 @@
+name="grive2-git"
+pkgname="grive2"
+pkgdir="${STOWDIR}/${name}"
+description="Google Drive client with support for new Drive REST API and partial sync"
+url="https://github.com/vitalif/grive2.git"
+build_depends="cmake libgcrypt20-dev libyajl-dev libboost-all-dev libcurl4-openssl-dev libexpat1-dev libcppunit-dev binutils-dev debhelper zlib1g-dev dpkg-dev pkg-config"
+depends="libgcrypt20 libyajl2 libboost-all-dev libcurl4 libexpat1"
+replace="grive"
+breaks="${pkgname} ${pkgname}-bin ${pkgname}-deb ${pkgname}-app"
+gives="grive2 grive"
```

Comment:
Yeah... This won't work, the CI's failing to build the dummy package. @Henryws & @D-Brox  summoned.

Reply:
Yeah you can't have two `gives`

Reply:
Actually I don't exactly understand the difference between pkgname and gives.
Anyway I set the gives value to the same name of pkgname.